message DseGuildCreate
{
	optional	int32	res		= 1;//结果
}

